src/main/java/racingcar/Winner.java

```diff
@@ -0,0 +1,29 @@
+package racingcar;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Winner {
+
+    private int findMaxPosition(List<Car> cars) {
+        return cars.stream()
+                .mapToInt(Car::getPosition)
+                .max()
+                .orElse(-1);
+    }
+
+    public String findWinner(List<Car> cars) {
+        int maxPosition = findMaxPosition(cars);
+
+        List<String> winnerList = cars.stream()
+                .filter(car -> car.getPosition() == maxPosition)
+                .map(car -> car.getName())
```

Comment:
메소드 레퍼런스 활용 해볼 수 있겠네요!

```suggestion
                .map(Car::getName)
```